smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/protocols/AbstractRestProtocol.java

```diff
@@ -660,113 +642,25 @@ private void updateResponsesMapWithResponseStatusAndObject(
-                    null, true));
-        }
-
-        // This map contains mapping of mediaType string (e.g., "application/json") to MediaTypeObject objects from
-        // conflicting error responses.
-        Map<String, List<MediaTypeObject>> contents = new LinkedHashMap<>();
-
-        // Combine conflicting error response objects into one response object.
-        ResponseObject.Builder combinedResponseBuilder = ResponseObject.builder();
-        for (ResponseObject error : errors) {
-            // Combine headers.
-            for (Map.Entry<String, Ref<ParameterObject>> header : error.getHeaders().entrySet()) {
-                combinedResponseBuilder.putHeader(header.getKey(), header.getValue());
-            }
-
-            // Collect contents from conflicting errors.
-            // E.g., if Error1 and Error2 both have application/json MediaTypeObject,
-            //  an entry in contents map will be ("application/json", [Error1/MediaTypeObject, Error2/MediaTypeObject]).
-            for (Map.Entry<String, MediaTypeObject> content : error.getContent().entrySet()) {
-                contents.computeIfAbsent(content.getKey(), (k) -> new ArrayList<>()).add(content.getValue());
-            }
-
-            // Combine links.
-            for (Map.Entry<String, Ref<LinkObject>> link : error.getLinks().entrySet()) {
-                combinedResponseBuilder.putLink(link.getKey(), link.getValue());
-            }
-        }
-
-        // Populate contents into union response.
-        for (Map.Entry<String, List<MediaTypeObject>> mapping : contents.entrySet()) {
-            if (mapping.getValue().size() == 1) {
-                combinedResponseBuilder.putContent(mapping.getKey(), mapping.getValue().get(0));
-            } else {
-                // Combine contents.
-                combinedResponseBuilder.putContent(mapping.getKey(), combineContents(mapping.getValue()));
-            }
-        }
-
-        // Set description, build, and add to responses.
-        combinedResponseBuilder.description("UnionError " + statusCode + " response");
-        responses.put(statusCode, combinedResponseBuilder.build());
-    }
-
-    private MediaTypeObject combineContents(List<MediaTypeObject> contents) {
-        MediaTypeObject.Builder result = MediaTypeObject.builder();
-        Schema.Builder schema = Schema.builder();
-        List<Schema> schemas = new ArrayList<>();
-        for (MediaTypeObject content : contents) {
-            schemas.add(content.getSchema().get());
-        }
-        return result.schema(schema.oneOf(schemas).build()).build();
-    }
-
     private ResponseObject createResponse(
             Context<T> context,
             HttpBindingIndex bindingIndex,
             EventStreamIndex eventStreamIndex,
             String statusCode,
             Shape operationOrError,
-            OperationShape operation,
-            boolean dropRequiredTrait
-    ) {
+            OperationShape operation
```

Comment:
Looks like `operation` was introduced in a later commit.

Reply:
Does it look like it should be removed?

Reply:
Looks like it's not being used in the method so it's probably fine.